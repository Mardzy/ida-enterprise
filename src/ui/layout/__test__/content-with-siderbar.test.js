import ContentWithSidebar from "../content-with-sidebar";

describe("The content with sidebar component", () => {

  it("renders its children", () => {
    const sidebar = () => <div className="sidebar">Sidebar</div>;
    const content = () => <div className="content">Content</div>;

    const wrapper = mount(
      <ContentWithSidebar sidebar={sidebar} content={content}/>
    );

    expect(toJson(wrapper)).toMatchSnapshot();
  });

});