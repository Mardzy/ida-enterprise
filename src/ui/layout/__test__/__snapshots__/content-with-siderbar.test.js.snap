// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The content with sidebar component renders its children 1`] = `
<ContentWithSidebar
  content={[Function]}
  sidebar={[Function]}
>
  <Grid
    columns={2}
    stackable={true}
  >
    <div
      className="ui stackable two column grid"
    >
      <GridColumn
        computer={3}
        tablet={5}
      >
        <div
          className="three wide computer five wide tablet column"
        >
          <div
            className="sidebar"
          >
            Sidebar
          </div>
        </div>
      </GridColumn>
      <GridColumn
        computer={13}
        tablet={11}
      >
        <div
          className="thirteen wide computer eleven wide tablet column"
        >
          <div
            className="content"
          >
            Content
          </div>
        </div>
      </GridColumn>
    </div>
  </Grid>
</ContentWithSidebar>
`;
